client/my-sites/checkout/checkout-thank-you/header.jsx

```diff
@@ -45,15 +25,12 @@ export class CheckoutThankYouHeader extends PureComponent {
 		hasFailedPurchases: PropTypes.bool,
 		isAtomic: PropTypes.bool,
 		isDataLoaded: PropTypes.bool.isRequired,
-		isSimplified: PropTypes.bool,
 		primaryCta: PropTypes.func,
 		primaryPurchase: PropTypes.object,
 		purchases: PropTypes.array,
 		recordTracksEvent: PropTypes.func.isRequired,
 		recordStartTransferClickInThankYou: PropTypes.func.isRequired,
 		selectedSite: PropTypes.object,
-		siteUnlaunchedBeforeUpgrade: PropTypes.bool,
```

Comment:
Let's also clear everything from components where we use it (unfortunately we don't have TS here to see errors ☹️)

https://github.com/Automattic/wp-calypso/blob/trunk/client/my-sites/checkout/checkout-thank-you/index.tsx#L725

Reply:
Ignore it, I am working around this place so I will clean it up